docker-compose.yml

```
@@ -41,4 +41,11 @@ services:
     stdin_open: true
     tty: true
     command: ["/bin/bash"]
-    
+
+  #This can help Docker automatically handle container failures and restart unhealthy containers.    
+  healthcheck:
+    test: ["CMD-SHELL", "curl -f http://localhost/ || exit 1"]
+    interval: 30s
+    timeout: 10s
+    retries: 3
+    start_period: 60s
```

indentaion level is as if it is a `service`. To apply this heathcheck to a `service` it must me indented just like `image` or `environmnet` is. The suggested code change below would apply the health check to the `app2` service
```suggestion
    # This can help Docker automatically handle container failures and restart unhealthy containers.    
    healthcheck:
      test: ["CMD-SHELL", "curl -f http://localhost/ || exit 1"]
      interval: 30s
      timeout: 10s
      retries: 3
      start_period: 60s
```